tests/system_tests_multicast.py

```diff
@@ -429,6 +449,245 @@ def test_999_check_for_leaks(self):
+        cls.INT_A.wait_router_connected('INT.C')
+        cls.INT_B.wait_router_connected('INT.A')
+        cls.INT_B.wait_router_connected('INT.C')
+        cls.INT_C.wait_router_connected('INT.A')
+        cls.INT_C.wait_router_connected('INT.B')
+        cls.EA1.wait_connectors()
+        cls.EB1.wait_connectors()
+        cls.EC1.wait_connectors()
+
+        # Client topology:
+        # all routes have 2 receivers
+        # Edge router EA1 and interior INT_A have a sender each
+        #
+        cls.config = [
+            # edge router EA1:
+            {'router':      cls.EA1,
+             'senders':     ['S-EA1-1'],
+             'receivers':   ['R-EA1-1', 'R-EA1-2'],
+             'subscribers': 2,
+             'remotes':     0
+             },
+            # Interior router INT_A:
+            {'router':      cls.INT_A,
+             'senders':     ['S-INT_A-1'],
+             'receivers':   ['R-INT_A-1', 'R-INT_A-2'],
+             'subscribers': 3,
+             'remotes':     1,
+             },
+            # Interior router INT_B:
+            {'router':      cls.INT_B,
+             'senders':     [],
+             'receivers':   ['R-INT_B-1', 'R-INT_B-2'],
+             'subscribers': 3,
+             'remotes':     1,
+             },
+            # edge router EB1
+            {'router':      cls.EB1,
+             'senders':     [],
+             'receivers':   ['R-EB1-1', 'R-EB1-2'],
+             'subscribers': 2,
+             'remotes':     0,
+             },
+            # Interior router INT_C:
+            {'router':      cls.INT_C,
+             'senders':     [],
+             'receivers':   ['R-INT_C-1', 'R-INT_C-2'],
+             'subscribers': 3,
+             'remotes':     1,
+             },
+            # edge router EC1
+            {'router':      cls.EC1,
+             'senders':     [],
+             'receivers':   ['R-EC1-1', 'R-EC1-2'],
+             'subscribers': 2,
+             'remotes':     0,
+             }
+        ]
+
+    def _get_alloc_stats(self, router, stats):
+        # return a map of the current allocator counters for each entity type
+        # name in stats
+
+        #
+        # 57: END = [{u'heldByThreads': int32(384), u'typeSize': int32(536),
+        # u'transferBatchSize': int32(64), u'globalFreeListMax': int32(0),
+        # u'batchesRebalancedToGlobal': int32(774), u'typeName':
+        # u'qd_buffer_t', u'batchesRebalancedToThreads': int32(736),
+        # u'totalFreeToHeap': int32(0), u'totalAllocFromHeap': int32(2816),
+        # u'localFreeListMax': int32(128), u'type':
+        # u'io.skupper.router.allocator', u'identity':
+        # u'allocator/qd_buffer_t', u'name': u'allocator/qd_buffer_t'}]
+
+        d = dict()
+        mgmt = router.management
+        q = mgmt.query(type=ALLOCATOR_TYPE).get_dicts()
+        for name in stats:
+            d[name] = next(a for a in q if a['typeName'] == name)
+
+        return d
+
+    def _check_for_leaks(self):
```

Comment:
This `_check_for_leaks(self)` can be removed since there is already code in alloc_pool.c that checks for leaks as seen here - 

https://github.com/skupperproject/skupper-router/blob/main/src/alloc_pool.c#L605

Reply:
Removed `_check_for_leaks(self)`